partner_email_check/models/res_partner.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from odoo import api, models, _
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from validate_email import validate_email
+except ImportError:
+    _logger.debug('Cannot import "validate_email".')
```

Comment:
Better `_logger.error` than `_logger.debug`, to show explicitly that this library is required. And define a validate_email method thats print `_logger.warning`, e.g.:
```
try:
    from validate_email import validate_email
except ImportError:
    _logger.error('Cannot import "validate_email".')
    def validate_email(email):
        _logger.warning("Can not validate email, python dependency required "validate_email")
        return True
```

Reply:
@zamberjo Thanks for your good advice, I fixed.